### PR TITLE
Add description for Donate page heading

### PR DESCRIPTION
**Pull request by nung22**
PolicyEngine_issue_699

### CODE REVIEW

**Comment by nung22** [comment]
Closes #669

**Comment by MaxGhenis** [comment]
You have to put `Fixes #669` in the original comment to link the issue. You can see it's not linked here
image

**Comment by nung22** [comment]
Thanks @MaxGhenis just linked it manually

File: src/redesign/components/Donate.jsx [hunk 1]
```diff
@@ -8,7 +8,10 @@ export default function Donate() {
   return (
     <div>
       <Header />
-      <PageHeader title="Donate" />
+      <PageHeader title="Donate">
+          Explore Brookings’ research and commentary to deepen your understanding of local, national, and global challenges.
```
Comment (nikhilwoodruff) [inline src/redesign/components/Donate.jsx:12]: Thanks but I think this was placeholder text in the Figma

**Pull request closed without merging**